extmod/ssl/__init__.py

```diff
@@ -0,0 +1 @@
+from .ssl import *
```

Comment:
This file is not doing anything, it's not in the manifest (and isn't really needed).